### ISSUE TITLE
Blue Shark / Sea Wolf - Controls Broken & Control options broken

### ISSUE THREAD

**Issue by M-Walrus** [MiSTer-devel/Arcade-SpaceInvaders_MiSTer#14]
Only way to control these games is with mouse by setting OSD "Gun Control: Disabled". Aiming is broken on any other setting.

Joy options do not allow control, nor does the mouse option. Mouse only works when set to "disabled", strangely enough.

**Comment by alanswx** [comment]
Did you setup your analog joystick correctly?

**Comment by M-Walrus** [comment]
yes, just double checked and analog is working with "joy1" selected in the OSD. Seems the core issue here is that the "mouse" and "disabled" options are flipped in the OSD for control.  Updating the issue to reflect that.
